chainer/variable.py

```diff
@@ -523,6 +536,12 @@ def __init__(self, data=None, **kwargs):
                     array_types[-1], type(data))
                 raise TypeError(msg)
 
+        self._init_attrs(data, name, grad, requires_grad, None)
+
+    def _init_attrs(self,
```

Comment:
Fix the position of line break. Either before `self` or before the parameter that does not fit the line.